app/frontend/components/pages/accounts/accounts.tsx

```diff
@@ -0,0 +1,286 @@
+import {Fragment, h} from 'preact'
+import {connect} from '../../../helpers/connect'
+import actions from '../../../actions'
+import range from '../../../../frontend/wallet/helpers/range'
+import printAda from '../../../helpers/printAda'
+import {Lovelace} from '../../../state'
+import {AdaIcon} from '../../common/svg'
+import Alert from '../../common/alert'
+import SendTransactionModal from './sendTransactionModal'
+import DelegationModal from './delegationModal'
+import tooltip from '../../common/tooltip'
+
+const Account = ({
```

Comment:
I would name this AccountTile or something like that, to distinguish it from the account object

Reply:
Agreed. :+1: